business/istio_validations.go

```diff
@@ -169,7 +169,7 @@ func (in *IstioValidationsService) GetIstioObjectValidations(ctx context.Context
 	go in.fetchRegistryServices(&registryServices, errChan, &wg)
```

Comment:
I see in the business/istio_validations.go that the logic is still fetching both Services and RegistryServices.
Is it something to consolidate later? I've just started to review the code top->down and that comes to my mind.

Reply:
I would leave that later, as services.Validations are used.